xrviz/dashboard.py

```diff
@@ -240,7 +248,8 @@ def create_plot(self, *args):
 
     def create_indexed_graph(self, *args):
         """
-        Creates graph for  selected indexes in selectors or players.
+        Creates a graph for the dimensions selected in `x` and `y`. This
```

Comment:
What is a graph, opposed to a plot?

Reply:
I don't think there is any difference. I will replace `create_plot` with `create_graph` to have similar function names.